units/host_termination_test.go

```diff
@@ -27,6 +28,12 @@ func init() {
 	}
 }
 
+func setupHostTerminationQueryIndex(t *testing.T) {
+	require.NoError(t, db.EnsureIndex(host.Collection, mongo.IndexModel{
```

Comment:
> index patterns have to be exact matches.

Do we need to add the second key here?

Reply:
Yes, because that index is the only index available.